django_sirene/importers.py

```diff
@@ -103,17 +122,11 @@ def _preload_data(self):
         """
         start = time.time()
 
-        self.db_activities_code = set(Activity.objects.values_list('code', flat=True))
-        self.db_legal_statuses_code = set(LegalStatus.objects.values_list('code', flat=True))
-        self.db_municipalities_code = set(Municipality.objects.values_list('code', flat=True))
-
-        qs = Institution.objects.values_list('id', 'siret', 'is_headquarter', 'is_expired')
-        for pk, siret, is_headquarter, is_expired in qs.iterator():
-                # take expired too to re-enable it if needed (deleted then created)
-                self.db_all_sirets.add(siret)
-                # but only headquarter not expired
-                if is_headquarter and not is_expired:
-                    self.db_headquarters[get_siren(siret)] = pk
```

Comment:
It seems like you could remove the definition of `self.db_headquarters` in the `__init__` method.